Update contact map info

kennethpdev opened the pull request:
~ fixed the width to take all the available space horizontally

vaarunh committed:
Initial commit

vaarunh committed:
Add files via upload

vaarunh committed:
Added Favicon

vaarunh committed:
Added Favicon

vaarunh committed:
Favicon

vaarunh committed:
Delete favicon.ico

vaarunh committed:
Update index.html

vaarunh committed:
Add files via upload

vaarunh committed:
Add files via upload

vaarunh committed:
Delete check.png

vaarunh committed:
Delete cursor.ico

vaarunh committed:
Add files via upload

vaarunh committed:
Update index.css

vaarunh committed:
Update index.html

vaarunh committed:
Add files via upload

vaarunh committed:
Delete index.css

vaarunh committed:
Update style.css

vaarunh committed:
Update style.css

vaarunh committed:
Add files via upload

ImgBotApp committed:
[ImgBot] Optimize images

/screenshot-varunherlekar.github.io-2021.09.09-14_20_22.png -- 596.85kb -> 396.57kb (33.56%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

vaarunh committed:
Merge pull request #1 from varunherlekar/imgbot

[ImgBot] Optimize images

vaarunh committed:
Update README.md

icoderharshit committed:
Fix: UI Issue #2

Updated the required styles in the stylesheet (style.css) and a few bootstrap breakpoints classes in html file (index.html) . Please, check @varunherlekar , This must resolve issue #2  and also a bit of some more quick UI/UX fixes to make the website more intuitive. Also, I have added the required comments in the stylesheet wherever needed.

vaarunh committed:
Merge pull request #3 from icoderharshit/Improve-UI-Footer

Fix: UI Issue #2 & some more fixes for better UI/UX

abee62 committed:
error message added for empty task

vaarunh committed:
Merge pull request #5 from abee62/main

error message added for empty task

kennethpdev committed:
Use flex when separating the todo text and todo actions

I think, taking advantage of the flex is a better option that float when separating nodes.

varunherlekar commented:
Thanks, I have added the label. Please check your Hacktober Fest Profile in 15-20mins. It will be updated

vaarunh committed:
Merge pull request #6 from kennethpdev/main

Use flex when separating the todo text and todo actions

ankitgauri26 committed:
=> fixed delete and mark as complete issue
=> created completed task list
=> breakdown the code into functions

vaarunh committed:
Merge pull request #8 from ankitgauri26/main

Completed task list

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: create .all-contributorsrc [skip ci]

vaarunh committed:
Merge pull request #10 from varunherlekar/all-contributors/add-ankitgauri26

docs: add ankitgauri26 as a contributor for code

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

vaarunh committed:
Merge pull request #11 from varunherlekar/all-contributors/add-kennethpdev

docs: add kennethpdev as a contributor for design, code

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

vaarunh committed:
Merge pull request #13 from varunherlekar/all-contributors/add-icoderharshit

docs: add icoderharshit as a contributor for design

ankitgauri26 committed:
=> updated the code for Edit task

vaarunh committed:
Update README.md

vaarunh committed:
Merge branch 'main' into all-contributors/add-abee62

vaarunh committed:
Update README.md

vaarunh committed:
Merge pull request #14 from ankitgauri26/main

Fix: Edit Task

vaarunh committed:
Merge pull request #12 from varunherlekar/all-contributors/add-abee62

docs: add abee62 as a contributor for code

Tarandeep-s1ngh committed:
Fix: UI Header and color pallet

Commit:
Added mark as important button

Tarandeep-s1ngh committed:
Fix: UI Header and color pallet

vaarunh committed:
Merge pull request #19 from Tarandeep-s1ngh/main

Style: UI Header fix and color pallet enhancement

vaarunh committed:
Merge pull request #18 from PRA-developer-TIK/pra-developer-tik

Added mark as important button

vaarunh committed:
Update .all-contributorsrc

vaarunh committed:
Delete .all-contributorsrc

vaarunh committed:
Update README.md

vaarunh committed:
Update README.md

gauri242 committed:
Update app.js

Fixes issue #22. Renamed Edit button to Save to improve user experience

ankitgauri26 committed:
Fixed issue - Feature to restore the completed task

jackwebdev committed:
Added "mark as important" functionality

- Add todo tasks to the top when marked as important.
- Style changes to todo when marked as important.

vaarunh committed:
Merge branch 'main' into main

vaarunh committed:
Merge pull request #26 from ankitgauri26/main

Fixed issue - Feature to restore the completed task

ImgBotApp committed:
[ImgBot] Optimize images

/Screenshot-main-app.png -- 406.23kb -> 331.00kb (18.52%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

vaarunh committed:
Merge pull request #30 from varunherlekar/imgbot

[ImgBot] Optimize images

The-Shivam-garg committed:
updation in footer

The-Shivam-garg committed:
update the footer

vaarunh committed:
Merge pull request #32 from The-Shivam-garg/shivam-addition

updation in footer

NitinNair89 committed:
Removed quotes api and background image

- Fixes #21
- Removed quotes JS and background image to allow better improvements and customizations in the future.
- Added few meta tags
- Corrected Bootstrap classes
- Set  default font-size for accessibility

NitinNair89 committed:
Merged changes from upstream

vaarunh committed:
Merge pull request #35 from NitinNair89/main

Removed quotes api and background image

jackwebdev committed:
Merge branch 'main' into main

vaarunh committed:
Merge pull request #28 from jackwebdev/main

Added "mark as important" functionality

vaarunh committed:
Update README.md

jackwebdev committed:
Added missing main tag and fixed footer position

jackwebdev committed:
Added missing main tag and fixed footer position

loud-mouth2 committed:
ADDED: Option to remove items once it has been marked completed.

shreyash-b committed:
.

vaarunh committed:
Merge pull request #40 from shreyash-b/main

Added persistence for task

vaarunh committed:
Merge pull request #39 from loud-mouth/main

ADDED: Option to remove items once it has been marked completed.

vaarunh committed:
Merge pull request #38 from jackwebdev/main

Added missing main tag and fixed footer position

gauri242 committed:
Merged changes from upstream

Fixed conflicts
Changed Edit button to Save

nikhil9302 committed:
strike-through added for completed tasks

vaarunh committed:
Merge pull request #43 from nikhil9302/main

strike-through added for completed tasks

vaarunh committed:
Merge pull request #42 from gauri242/main

Fixed issue #22

vaarunh committed:
Revert "Fixed issue #22"

vaarunh committed:
Merge pull request #45 from varunherlekar/revert-42-main

Revert "Fixed issue #22"

vaarunh committed:
Revert "strike-through added for completed tasks"

vaarunh committed:
Merge pull request #46 from varunherlekar/revert-43-main

Revert "strike-through added for completed tasks"

loud-mouth committed:
ADDED: Single Function for OnClick Buttons for Completed Tasks

vaarunh committed:
Merge pull request #48 from loud-mouth/main

ADDED: Single Function for OnClick Buttons for Completed Tasks

shreyash-b committed:
fixed remove completed task bug

vaarunh committed:
Merge pull request #52 from shreyash-b/main

Fix remove completed task bug

donjosemathew committed:
H1 Changes

vaarunh committed:
Merge pull request #57 from dontech09/main

H1 Changes

manognyaa committed:
UI overview for the task page

vaarunh committed:
Merge pull request #58 from manognyaa/main

UI overview for the task page

ankitgauri26 committed:
Validating the user input while editing value

vaarunh committed:
Merge pull request #59 from ankitgauri26/main

Validating the user input while editing value

ronzano2 committed:
showing action buttons on hover

vaarunh committed:
Merge pull request #61 from ba2sik/main

Showing actions buttons only on hover

anshu189 committed:
Added Design Part

vaarunh committed:
Merge pull request #62 from anshu189/main

Added User Friendly Modern Design

shreyadenny committed:
added icons to the buttons

shreyadenny committed:
Add files via upload

vaarunh committed:
Merge pull request #65 from shreyadenny/main

Added icons to the buttons

kennethpdev committed:
Fix action buttons in mobile viewport

~ added margins and remove flex shrink that has no effect at all
